internal/monitor/features/integration.feature

```diff
@@ -657,3 +810,23 @@ Feature: Integration Test
       | ""         | "1-3"       | "2-2" | "0-0" | "unity"    | "unity-nfs"    | "one-third" | "zero"  | "driverpod" | "offline.unity.storage.dell.com" | 120      | 300        | 300     | 600           |
       | ""         | "1-2"       | "1-1" | "0-0" | "unity"    | "unity-iscsi"  | "one-third" | "zero"  | "driverpod" | "offline.unity.storage.dell.com" | 120      | 300        | 300     | 600           | 
       | ""         | "1-3"       | "2-2" | "0-0" | "unity"    | "unity-iscsi"  | "one-third" | "zero"  | "driverpod" | "offline.unity.storage.dell.com" | 120      | 300        | 300     | 600           |
+
+@powerstore-integration
+  Scenario Outline: Basic node failover testing using test StatefulSet pods (driver pods down)
+    Given a kubernetes <kubeConfig>
+    And cluster is clean of test pods
+    And wait <nodeCleanSecs> to see there are no taints
+    And <podsPerNode> pods per node with <nVol> volumes and <nDev> devices using <driverType> and <storageClass> in <deploySecs>
+    Then validate that all pods are running within <deploySecs> seconds
+    When I fail <workers> worker driver pod with <failure> failure for <failSecs> and I expect these taints <taints>
+    And the taints for the failed nodes are removed within <nodeCleanSecs> seconds
+    Then finally cleanup everything
+
+    Examples:
+      | kubeConfig | podsPerNode | nVol  | nDev  | driverType    | storageClass          | workers     | primary | failure     |  taints                               | failSecs | deploySecs | runSecs | nodeCleanSecs |
+      | ""         | "1-2"       | "1-1" | "0-0" | "powerstore"  | "powerstore-nfs"      | "one-third" | "zero"  | "driverpod" | "offline.powerstore.storage.dell.com" | 120      | 300        | 300     | 600           | 
+      | ""         | "1-3"       | "2-2" | "0-0" | "powerstore"  | "powerstore-nfs"      | "one-third" | "zero"  | "driverpod" | "offline.powerstore.storage.dell.com" | 120      | 300        | 300     | 600           |
+      | ""         | "1-2"       | "1-1" | "0-0" | "powerstore"  | "powerstore-iscsi"    | "one-third" | "zero"  | "driverpod" | "offline.powerstore.storage.dell.com" | 120      | 300        | 300     | 600           | 
+      | ""         | "1-3"       | "2-2" | "0-0" | "powerstore"  | "powerstore-iscsi"    | "one-third" | "zero"  | "driverpod" | "offline.powerstore.storage.dell.com" | 120      | 300        | 300     | 600           |
+      #| ""         | "1-2"       | "1-1" | "0-0" | "powerstore"  | "powerstore-nvmetcp"  | "one-third" | "zero"  | "driverpod" | "offline.powerstore.storage.dell.com" | 120      | 300        | 300     | 600           | 
```

Comment:
same as above

Reply:
As we have to run e2e & longevity tests via csm-pipeline which doesn't support nvme-tcp, we have commented this for now. Manual testing with nvme-tcp has been done.